packages/datetime2/src/index.ts

```diff
@@ -27,25 +27,28 @@ export { Classes as Datetime2Classes, ReactDayPickerClasses } from "./classes";
 /* eslint-disable deprecation/deprecation */
 
 export {
-    /** @deprecated import from `@blueprintjs/datetime` or use `Datetime2Classes` instead */
+    /** @deprecated use `Datetime2Classes` instead */
     Classes,
-    type DateFormatProps,
-    /** @deprecated import from `@blueprintjs/datetime` instead */
     DateInput as DateInput2,
-    /** @deprecated import from `@blueprintjs/datetime` instead */
+    /** @deprecated use `DateInput3Props` instead */
```

Comment:
I would be in support of improving this deprecation message with an "or preferably use ___3"

Reply:
just changed it back to the old one